templates/blog/article.html

```diff
@@ -32,111 +34,240 @@
 {% endblock %}
 
 {% block content %}
+<div class="u-fixed-width">
+  <nav class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
```

Comment:
Out of curiosity, why would this breadcrumb need slightly more padding below and others don't?

Reply:
This should be using the breadcrumbs partial for consistency

Reply:
I've created an issue on vanilla to fix this, and removed the utility class. Can I put this patch and TODO comment in the pr for now:
```// XXX 03/07/23 Lyubomir Popov - https://github.com/canonical/vanilla-framework/issues/4818
.p-breadcrumbs {
  margin-bottom: $spv--small;
}
```

Reply:
@ClementChaumel how will this logic work with the breadcrumbs partial? 
`
          {% if "display_category" in article and article.display_category.name %}
          {% if article.display_category.name == 'Articles' %}
            Article
          {% elif article.display_category.name == 'Case studies' %}
            Case study
          {% elif article.display_category.name == 'News' %}
            News
          {% elif article.display_category.name == 'Webinars' %}
            Webinar
          {% elif article.display_category.name == 'White papers' %}
            White paper
          {% else %}
            {{ article.display_category.name }}
          {% endif %}
        {% else %}
          Article
        {% endif %}`

Reply:
seems to work, it just says blog/ Article. I think we can use it like so for now, then if someone wants to improve the partial to be category-specific for blogs, that can be a future improvement?